### PR TITLE
Enhanced docblock for tribe_get_listview_past_link function

### PR DESCRIPTION
**Pull request by Sjaure**
- Added a detailed description of the function's parameters and return value.

### 🎫 Ticket

[TICKET_ID]

### ✔️ Checklist
- [ ] Changelog entry in the `readme.txt` file.
- [ ] Code is covered by **NEW** `wpunit` or `integration` tests.
- [ ] Code is covered by **EXISTING** `wpunit` or `integration` tests.
- [ ] Are all the **required** tests passing?
- [ ] Automated code review comments are addressed.
- [ ] Have you added Artifacts?
- [ ] Check the base branch for your PR.
- [ ] Add your PR to the project board for the release.

### CODE REVIEW

File: src/functions/template-tags/link.php [hunk 1]
```diff
@@ -196,13 +196,13 @@ function tribe_get_listview_link( $term = null ) {
...
+ * Returns a link to the general or category past view.
  *
- * Returns a link to the general or category past view
+ * @since  5.14.0
  *
- * @param int|null $term Term ID
+ * @param int|null $term Optional. Term ID. Default null.
  *
- * @return string URL
+ * @return string                  URL of the past list view.
```
Comment (JPry) [inline src/functions/template-tags/link.php:205]: The `@return` comment shouldn't align with the `@param` comments.

```suggestion
 * @return string URL of the past list view.
```

Comment (dpanta94) [inline src/functions/template-tags/link.php:205]: Why all that space ? I would keep the space if there was no blank line between the items

**Review by dpanta94 — CHANGES_REQUESTED** [review]
How about we add a changelog using `npm run changelog` ? I think its a good idea to start using it

File: src/functions/template-tags/link.php [hunk 2]
```diff
@@ -196,13 +196,13 @@ function tribe_get_listview_link( $term = null ) {
 }
 
 /**
- * Link to List View (Past)
+ * Returns a link to the general or category past view.
  *
- * Returns a link to the general or category past view
+ * @since  5.14.0
```
Comment (JPry) [inline src/functions/template-tags/link.php:201]:
```suggestion
 * @since 5.14.0
```

**Comment by codingmusician** [comment]
Do we have a current ticket for al lthe documentation work that has been done? If so, let's link it. Also, aren't we adding changelogs for docs, too?

File: src/functions/template-tags/link.php [hunk 3]
```diff
@@ -202,13 +202,13 @@ function tribe_get_listview_link( $term = null ) {
 }
 
 /**
- * Link to List View (Past)
+ * Returns a link to the general or category past view.
  *
- * Returns a link to the general or category past view
+ * @since 5.14.0
```
Comment (bordoni) [inline src/functions/template-tags/link.php:207]: This since here is wrong, this method exists before `5.14.0`.

**Review by bordoni — CHANGES_REQUESTED** [review]

File: src/functions/template-tags/link.php (hunk 3 above)
Comment (Camwyn) [reply src/functions/template-tags/link.php:207]: 2.0.1

Comment (Sjaure) [reply src/functions/template-tags/link.php]: @bordoni , this is fixed

File: src/functions/template-tags/link.php [hunk 4]
```diff
@@ -202,13 +202,13 @@ function tribe_get_listview_link( $term = null ) {
 }
 
 /**
- * Link to List View (Past)
+ * Returns a link to the general or category past view.
```
Comment (Camwyn) [inline src/functions/template-tags/link.php:205]:
```suggestion
 * Returns a url to the general or category past view.
```
There is confusion some times about link = url vs. link = html anchor tag

**Review by Camwyn — CHANGES_REQUESTED** [review]

**Review by Camwyn — DISMISSED** [review]

**Review by pattihis — APPROVED** [review]

**Review by codingmusician — APPROVED** [review]

**Pull request merged**